modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/configuration/user_settings.jspf

```diff
@@ -38,7 +38,7 @@
 </aui:fieldset>
 
 <aui:script>
-	const usePortalTimeZoneCheckbox = document.getElementById(
+	var usePortalTimeZoneCheckbox = document.getElementById(
```

Comment:
Hey @rijema, why do we need to define this as a var instead of keeping it as conts?

Reply:
We were having this error:

 Uncaught SyntaxError: Identifier 'usePortalTimeZoneCheckbox' has already been declared
 
 Because the checkbox in the first page (User settings) in the same modal we're being called twice.

Reply:
I see, thanks for the explanation!